pkg/providers/vsphere/testdata/cluster_main_multiple_worker_node_groups.yaml

```diff
@@ -23,11 +23,6 @@ spec:
         name: test-wn
         kind: VSphereMachineConfig
       name: md-1
-  externalEtcdConfiguration:
-    count: 3
-    machineGroupRef:
-      name: test-etcd
-      kind: VSphereMachineConfig
```

Comment:
why are we changing this? :)

Reply:
I'm using this file for multiple tests & it required more configurations for the tests outside of the scope of the feature that we're actually testing so I removed it